apps/ledger-live-desktop/src/renderer/screens/account/index.tsx

```diff
@@ -109,12 +121,28 @@ const AccountPage = ({
     [hiddenNftCollections, shouldFilterTokenOpsZeroAmount],
   );
 
+  const currency = getAccountCurrency(account);
+
+  useEffect(() => {
+    try {
```

Comment:
I assume that we don't need to use hooks(useEffect and the useState for currencyConfig) for currencyConfig.
we can just use a variable "currencyConfig"
```
const currencyConfig = getCurrencyConfiguration(currency);
```